apps/react-three-org/src/app.tsx

```diff
@@ -4,30 +4,25 @@ import { useQuery } from '@tanstack/react-query'
 import { PackageCard } from '@/components/package-card'
 import { ProjectConfigurator } from '@/components/project-configurator'
 import { NavBar } from '@/components/nav-bar'
-import { packages } from '@/lib/packages'
+import { packages, tools, PackageIDs, ToolIDs } from '@/lib/packages'
 import { BackgroundAnimation } from '@/components/background-animation'
-import { Button } from './components/ui/button.js'
-import { CheckSquare, PackageIcon, X } from 'lucide-react'
+import { CogIcon, PackageIcon } from 'lucide-react'
 import { Toaster } from 'sonner'
+import { SelectionSection } from './components/selection-section.js'
 
 const searchParams = new URLSearchParams(location.search)
-
 const sessionAccessTokenKey = 'access_token'
 const sessionAccessToken = sessionStorage.getItem(sessionAccessTokenKey)
 
 export function App() {
   const [state, setState] = useState(() => searchParams.get('state'))
-  const [selectedPackages, setSelectedPackages] = useState<string[]>([])
+  const [selectedPackages, setSelectedPackages] = useState<PackageIDs[]>([])
```

Comment:
Currently the CLI defaults to everything selected, where as the app defaults to nothing selected. 

Should there be a sensible middle ground?